docs/powershell.ps1

```diff
@@ -0,0 +1,21 @@
+# Powershell script to configure SSH on vSphere Windows VMs
```

Comment:
Please update this script so that it matches the current user data.

Reply:
https://issues.redhat.com/browse/WINC-533
Discussed elsewhere, this will be done when we update the CI image to use the current user data ssh method.

Reply:
was this addressed?